File: tests/fixtures/edgecase_repo/python/pkg/module7.py
"""Module 7 of the generated Python layer."""

from __future__ import annotations

from pkg.module6 import Service6


class Base7:
    """Base class for service 7."""


class Service7(Base7):
    """Service 7."""

    def handle(self, payload):
        """Handle a payload."""
        self.validate(payload)
        return Service6().handle(payload)

    def validate(self, payload):
        """Validate a payload."""
        return payload is not None


def helper_7(value):
    """Module-level helper."""
    return value

File: tests/fixtures/edgecase_repo/go/module7/module7.go
package module7

import (
	"fmt"

	"example.com/pkg/module6"
)

type Base7 struct {
	Name string
}

type Service7 struct {
	Base7
}

type Handler7 interface {
	Handle(payload string) bool
}

func (s *Service7) Handle(payload string) bool {
	s.validate(payload)
	fmt.Println(payload)
	return true
}

func (s *Service7) validate(payload string) {
}

func Helper7(value string) string {
	_ = module6.Helper6("x")
	return value
}
